Content.Server/Salvage/SpawnSalvageMissionJob.cs

```diff
@@ -349,7 +376,7 @@ private async Task SpawnMobsRandomRooms(SalvageMission mission, Dungeon dungeon,
 
             foreach (var group in faction.MobGroups)
             {
-                value += group.Prob;
+                value += group.Prob * scale;
```

Comment:
You should multiply groupSpawns by the scale instead IMO.

I also want to bump mob count a fair bit on the lower end at some point.